file-formats/susi/examples/55fae0107acd2ee619d9f869c651edhs.susi.json

```diff
@@ -29,6 +29,9 @@
         {
           "field": "PROGRAM",
           "low": "SAPLSTRF"
+        },
+        {
+          "field": "FILENAME"
```

Comment:
Maybe, in your case it make sense to mark the "low" value as `$showalways`

Reply:
"low" is marked as `$showAlways`, so I think this is a mistake in the example

Reply:
I have just fixed the example with https://github.com/SAP/abap-file-formats/pull/526/commits/5b381aa435c82906849c0463b1cf43ca8ac84fc0

Reply:
Katharina is right, I corrected the example